mender-convert-package

```diff
@@ -150,8 +150,9 @@ temp_img_name=$(basename "$disk_image")
 
 # Strip the image suffix from temp_img_name and set the original image name as
 # the output image name
-image_name="${temp_img_name%.*}-${device_type}-mender"
 
+# replaces whitespace with - to seperate device types. Makes it possible to create files with device_type in the filename   
+image_name="${temp_img_name%.*}-$(sed 's| |+|g' <<< $device_type)-mender"
```

Comment:
Nitpick:
There is 5 spaces at the end of `filename` in the comment.

Also, now the comment above:
`Strip the image suffix`
is left dangling in the air, with no command to decorate.